dali/util/std_cufile.cc

```diff
@@ -115,29 +115,39 @@ void StdCUFileStream::HandleIOError(int64 ret) const {
   }
 }
 
-size_t StdCUFileStream::ReadGPU(uint8_t* gpu_buffer, size_t n_bytes, size_t offset) {
+size_t StdCUFileStream::ReadGPUThread(uint8_t* gpu_buffer, size_t n_bytes,
```

Comment:
nitpick: I'd call this `ReadGPUImpl` or similar, since it doesn't contain anything thread specific.

Reply:
fixed